test-project/src/test/java/org/liquibase/ext/couchbase/plugin/common/TestContainerInitializer.java

```diff
@@ -0,0 +1,107 @@
+package org.liquibase.ext.couchbase.plugin.common;
+
+import liquibase.ext.couchbase.database.CouchbaseLiquibaseDatabase;
+import lombok.SneakyThrows;
+import org.liquibase.ext.couchbase.plugin.containers.JavaMavenContainer;
+import org.liquibase.ext.couchbase.plugin.util.TestPropertyProvider;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.couchbase.BucketDefinition;
+import org.testcontainers.couchbase.CouchbaseContainer;
+import org.testcontainers.couchbase.CouchbaseService;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+
+public class TestContainerInitializer {
+    private static final String COUCHBASE_IMAGE_NAME = TestPropertyProvider.getProperty("couchbase.image.name");
+    private static final String COUCHBASE_IMAGE_VERSION = TestPropertyProvider.getProperty("couchbase.version");
+    private static final String TEST_PROJECT_ABSOLUTE_PATH = System.getProperty("user.dir");
+    private static final String TEST_PROJECT_MAIN_RESOURCES_PATH = "/test-project/src/main/resources/";
+    public static final String ROOT_WITH_TEST_PROJECT_TEST_RESOURCES = TEST_PROJECT_ABSOLUTE_PATH + "/src/test/resources/";
+    private static final String COUCHBASE_NETWORK_ALIAS = "couchbase";
+    private static final String LIQUIBASE_COUCHBASE_PROJECT_BASE_DIR = "/liquibase-couchbase-extension";
+    private static final String LIQUIBASE_PROPERTIES_FILE = "liquibase.properties";
+    private static final String DEPENDENCIES_VOLUME = "/dependencies";
```

Comment:
I propose to add more static constants here as some string values are used many times in this class. For example `/root/.m2/repository` and others. Not necessary to move them all but at least values that are used few times

Reply:
Done.